composer.json

```diff
@@ -43,28 +43,38 @@
         "symfony/phpunit-bridge": "^5.3",
         "symfony/polyfill-uuid": "^1.0"
     },
+    "conflict": {
+        "nette/php-generator": "3.6.3",
```

Comment:
unforuntately, this conflict applies when this package is required, but this is only a dev dep, unecessarily preventing the install of this dep.